scripts/README.md

```diff
@@ -55,6 +55,9 @@ In case of a problem, another method to generate the changelog is available at t
 3. `pnpm release:publish` (release the versions on npm, you need your 2FA device)
 4. `pnpm release:tag` (push the newly created tag)
 
+> [!WARNING]
+> If the `pnpm release:tag` fails you can create and push the tag using the following command: `git tag -a v4.0.0-alpha.30 -m "Version 4.0.0-alpha.30" && git push upstream --tag`.
```

Comment:
```suggestion
> If the `pnpm release:tag` fails you can create and push the tag using the following command: `git tag -a v4.0.0-alpha.30 -m "Version 4.0.0-alpha.30" && git push upstream --tag`.
> Make sure to copy the git tag command above so that the tag is annotated!
```